### PR TITLE
[New Terms] Java Math Methods: `.max()` and `.min()`

### LINKED ISSUE #903: [Term Entries] Java Math Methods: `.max()` and `.min()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.max()` and `Math.min()` Java methods.

- In the `terms/` folder, make new folders called `max` and `min`.
- Inside the `max` and `min` folders, create a new markdown file called `max.md` and `min.md`, respectively.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by MinalGhate**
### Description
closes #903  
- Made two folders max and min inside content/java/concepts/math-method/terms/ and inside that added file max.md and min.md respectively.
- Inside max.md added explanation of math.max() function according to entry template, added syntax and example demonstrating it.
- Similarly inside min.md file added explanation of math.min() function, added syntax and example demonstrating it. 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@MinalGhate, I did a first review and left a few comments to address.

**Comment by MinalGhate** [comment]
done 👍

File: content/java/concepts/math-methods/terms/max/max.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+---
+
+The **`Math.max()`** method returns the maximum value from the given two arguments.
+
+## Syntax
+
+```pseudo
+Math.max(num1, num2)
+```
+- The `.max()` function takes two parameters `num1` and `num2`.
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/max/max.md:22]:
```suggestion

- The `.max()` method takes two parameters `num1` and `num2`.
```

File: content/java/concepts/math-methods/terms/max/max.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- The `num1` and `num2` arguments can be of type int, float, double or long.
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/max/max.md:23]: link type to: https://www.codecademy.com/resources/docs/java/data-types

Comment (yangc95) [reply content/java/concepts/math-methods/terms/max/max.md:23]: Also, maybe let's place the types in backticks. Can we also do this for the other entry

File: content/java/concepts/math-methods/terms/max/max.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        int num2 = 10;
+
+        System.out.println(Math.max(num1, num2));
+    }
+}
+```
+
+This results in the following output:
+
+```java
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/max/max.md:42]:
```suggestion
```shell
```

File: content/java/concepts/math-methods/terms/min/min.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+---
+
+The **`Math.min()`** method returns the minimum value from the given two arguments.
+
+## Syntax
+
+```pseudo
+Math.min(num1, num2)
+```
+- The `.min()` function takes two parameters `num1` and `num2`.
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/min/min.md:22]:
```suggestion

- The `.min()` method takes two parameters `num1` and `num2`.
```

File: content/java/concepts/math-methods/terms/min/min.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        int num2 = 10;
+
+        System.out.println(Math.min(num1, num2));
+    }
+}
+```
+
+This results in the following output:
+
+```java
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/min/min.md:42]:
```suggestion
```shell
```

**Review by yangc95 — COMMENTED** [review]
Hi @MinalGhate! I conducted another review. Please tag me once you've looked over the comments or have any questions regarding them.

File: content/java/concepts/math-methods/terms/max/max.md (hunk 2 above)
Comment (MinalGhate) [reply content/java/concepts/math-methods/terms/max/max.md:24]: Hey @yangc95 
I didn't understand what I have to do in this, do I have to make "type" word clickable which will redirect to https://www.codecademy.com/resources/docs/java/data-types ?

**Comment by MinalGhate** [comment]
Hey @yangc95
Can I add one more example where we can find maximum of more than two number using this method (by Math.max(num1,Math.max(num2,num3)) ), or may be can I add by extending existing example.

File: content/java/concepts/math-methods/terms/max/max.md (hunk 2 above)
Comment (yangc95) [reply content/java/concepts/math-methods/terms/max/max.md:24]: Hi @MinalGhate,

Yes! Exactly~

**Comment by yangc95** [comment]
> Hey @yangc95
> Can I add one more example where we can find maximum of more than two number using this method (by Math.max(num1,Math.max(num2,num3)) ), or may be can I add by extending existing example.

@MinalGhate sure! If you'd like to create a code byte example that would be great

**Comment by MinalGhate** [comment]
Hey @yangc95 
done 👍

File: content/java/concepts/math-methods/terms/max/max.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+The **`Math.max()`** method returns the maximum value from the given two arguments.
+
+## Syntax
+
+```pseudo
+Math.max(num1, num2)
+```
+
+- The `.max()` method takes two parameters `num1` and `num2`.
+- The `num1` and `num2` arguments can be of <a href ="https://www.codecademy.com/resources/docs/java/data-types"> type </a> int, float, double or long.
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/max/max.md:24]: Liking the use of anchor tags but let's write in markdown format like:

```suggestion
- The `num1` and `num2` arguments can be of [type](https://www.codecademy.com/resources/docs/java/data-types) int, float, double or long.
```

**Comment by MinalGhate** [comment]
Hey @yangc95
Done 👍

**Review by yangc95 — APPROVED** [review]

**Comment by Dusch4593** [comment]
Thank you so much @MinalGhate for your contributions to Codecademy Docs! Your published entries can be viewed below.

- `.max()`: https://www.codecademy.com/resources/docs/java/math-methods/max
- `.min()`: https://www.codecademy.com/resources/docs/java/math-methods/min

**Pull request merged**